django/template/base.py

```diff
@@ -757,6 +757,11 @@ def __init__(self, var):
             raise TypeError(
                 "Variable must be a string or number, got %s" % type(var))
         try:
+            # Python treats 'inf' as a float. We don't want to treat it as
```

Comment:
Avoid use of "we" in comments per our [style guide](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).